### ISSUE TITLE
Character Limit Lengths

### ISSUE THREAD

**Issue by MichaelDalrymple** [wet-boew/wet-boew#7870]
A note that on http://wet-boew.github.io/wet-boew-styleguide/v4/writing/strctr-en.html#image, it states that alt text should be under 125 characters. However the Canada.ca style guide states 140. I suspect the Style Guide takes precedence in these matters, so this may need to be updated.

**Comment by crochefort** [comment]
you should open a ticket here instead : https://github.com/wet-boew/wet-boew-styleguide
